packages/packages/core/editor-variables/src/hooks/use-permissions.ts

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react';
+import { useCurrentUserCapabilities } from '@elementor/editor-current-user';
+
+export const usePermissions = () => {
```

Comment:
@mike-elementor, I think you should be more specific

```suggestion
export const useVariablesPermissions = () => {
```